Change look of home page, 404 page headings. closes #2385

webinista opened the pull request:
**Blocked by #2422. Merge that first.**
- Changing class names for index, 404 pages.
- Removing #content .splash h2 selector
- Removing .pitch CSS block.
- Simplifying .splash--app h2 selector.
- Adjusting margin on .va-tagline--home
- Changing class names for index, 404 pages.
- Removing redundant #content .splash styles.
- Simplifying #content .splash to just .splash

webinista committed:
Change look of home page, 404 page headings. closes #2385

- Changing class names for index, 404 pages.
- Removing #content .splash h2 selector
- Removing .pitch CSS block.
- Simplifying .splash--app h2 selector.
- Adjusting margin on .va-tagline--home
- Changing class names for index, 404 pages.
- Removing redundant #content .splash styles.
- Simplifying #content .splash to just .splash

webinista committed:
Change look of home page, 404 page headings. closes #2385

- Changing class names for index, 404 pages.
- Removing #content .splash h2 selector
- Removing .pitch CSS block.
- Simplifying .splash--app h2 selector.
- Adjusting margin on .va-tagline--home
- Changing class names for index, 404 pages.
- Removing redundant #content .splash styles.
- Simplifying #content .splash to just .splash

webinista committed:
Change look of home page, 404 page headings. closes #2385

- Changing class names for index, 404 pages.
- Removing #content .splash h2 selector
- Removing .pitch CSS block.
- Simplifying .splash--app h2 selector.
- Adjusting margin on .va-tagline--home
- Changing class names for index, 404 pages.
- Removing redundant #content .splash styles.
- Simplifying #content .splash to just .splash

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

angel commented:
Can we add more padding above the "Please try one..." so it's easier to read? 

Something like this: 
![image](https://cloud.githubusercontent.com/assets/103496/15721547/572922e6-27ee-11e6-85e8-511681d79866.png)

Thanks!

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'home-banner' of github.com:department-of-veterans-affairs/vets-website into home-banner

webinista committed:
Adding space above the 404 page subheading.

Also refactored and/or removed redundant or problematic CSS

- Removed vertical padding from .primary.
- Removed styles for #content .section h3.
- Removed bottom padding from h3.
- Removed top margin from .primary h3.
- Removed .home class from 404 page.
- Removed .fourohfour .navigation--major selector.
- Removed .home #content h3 selector, styles.
- Reduced bottom margin on .va-quicklinks h3.

webinista committed:
Adding space above the 404 page subheading.

Also refactored and/or removed redundant or problematic CSS

- Removed vertical padding from .primary.
- Removed styles for #content .section h3.
- Removed bottom padding from h3.
- Removed top margin from .primary h3.
- Removed .home class from 404 page.
- Removed .fourohfour .navigation--major selector.
- Removed .home #content h3 selector, styles.
- Reduced bottom margin on .va-quicklinks h3.

Commit:
Merge branch 'master' into home-banner

emilyville commented:
This looks good to me! It's a "big" change, so going to merge today once I make sure I am not surprising anyone.

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

crwallace committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'home-banner' of github.com:department-of-veterans-affairs/vets-website into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista committed:
Merge branch 'master' into home-banner

webinista commented:
Closing since the 404 page went in a different direction.
